Makefile

```diff
@@ -376,11 +380,11 @@ db/teardown:
 
 .PHONY: mqtt/prepare
 mqtt/prepare:
-	@echo $(shell LC_CTYPE=C && cat /dev/urandom | tr -dc 'a-zA-Z0-9' | head -c 13) > $(mqtt_password_file)
+	@echo $(shell LC_CTYPE=C tr -dc 'a-zA-Z0-9' < /dev/urandom | head -c 13) > $(mqtt_password_file)
 
 .PHONY: mqtt/setup
 mqtt/setup: mqtt/prepare
-	@echo '{"brokerHost": "localhost:1883", "username": "$(mqtt_user)", "password": "$(shell cat $(mqtt_password_file))"}' > $(mqtt_config_file)
+	@echo '{"brokerHost":"localhost:1883","username":"$(mqtt_user)","password":"$(shell cat $(mqtt_password_file))","topics":{"sourceEvents":"sources/maestro-integration/consumers/+/sourceevents","agentEvents":"sources/maestro-integration/consumers/+/agentevents"}}' > $(mqtt_config_file)
```

Comment:
should't we use `$(source_id)`, why hard code `maestro-integration` here?

Reply:
I think this only for integration test, the source id is always `maestro-integration` in  integration test

Reply:
move this config under the `test-integration` task, it may be more better